hybrid/SampleApps/NoteSync/NoteSync.xcodeproj/project.pbxproj

```diff
@@ -645,9 +652,10 @@
 /* End PBXSourcesBuildPhase section */
 
 /* Begin PBXTargetDependency section */
-		CE4CE4D21C0E6838009F6029 /* PBXTargetDependency */ = {
+		829DA2781C1263260040F5F1 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
 			name = SalesforceHybridSDK;
+			targetProxy = 829DA2771C1263260040F5F1 /* PBXContainerItemProxy */;
```

Comment:
This is another common "missing" item before.  Probably links with the new configuration mentioned above?